modules/machine-health-checks-about.adoc

```diff
@@ -11,11 +11,6 @@ Machine health checks automatically repair unhealthy machines in a particular ma
 
 To monitor machine health, create a resource to define the configuration for a controller. Set a condition to check, such as staying in the `NotReady` status for five minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
 
-[NOTE]
-====
-You cannot apply a machine health check to a machine with the master role.
-====
-
```

Comment:
Is this still helpful for those not using CPMS?

Reply:
mmmmm good point, I will refine rather than removing.